sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/README.md

```diff
@@ -0,0 +1,81 @@
+
+# Microsoft Azure MobileNetwork management client library for .NET
+
+[Azure Private 5G](https://learn.microsoft.com/en-us/azure/private-5g-core/private-5g-core-overview) is an Azure cloud service for deploying and managing 5G core network functions on an Azure Stack Edge device, as part of an on-premises private mobile network for enterprises. The 5G core network functions connect with standard 4G and 5G standalone radio access networks (RANs) to provide high performance, low latency, and secure connectivity for 5G Internet of Things (IoT) devices. Azure Private 5G Core gives enterprises full control and visibility of their private mobile networks.
```

Comment:
Please remove `en-us` in the `Azure Private 5G` doc link. It's required by [Complianc CI](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2112415&view=logs&j=bc67675d-56bf-581f-e0a2-208848ba68ca&t=480ecc96-5c8f-5bc8-1dbc-f6f32b953559&l=22).